README.md

```diff
@@ -32,6 +32,8 @@ Options:
       --chrono-dates     Use chrono::NaiveDateTime for date/timestamps logical types
       --derive-builders  Derive builders for generated record structs
       --derive-schemas   Derive AvroSchema for generated record structs
+      --extra_derive    Append one extra derive macros to the generated record structs
```

Comment:
```suggestion
      --extra_derive     Append one extra derive macros to the generated record structs
```

Reply:
IMO there is no need of `--extra_derive`.
`--extra_derives` should be enough for everyone.